### PR TITLE
Add tutorials for different frameworks

### PR DESCRIPTION
**Pull request by fifoosid**

### CODE REVIEW

**Comment by fifoosid** [comment]
Fixes #538

File: docs/React-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Get started with UI5 Web Components & React
+
+In this tutorial, we are going to show how to get add UI5 Web Components in your application. UI5 Web Components can be added to both new React application as well as already existing one.
```
Comment (vladitasev) [inline docs/React-tutorial.md]: same as above

File: docs/Angular-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Get started with UI5 Web Components & Angular
+
+In this tutorial, we are going to show how to get add UI5 Web Components in your application. UI5 Web Components can be added to both new Angular application as well as already existing one.
```
Comment (vladitasev) [inline docs/Angular-tutorial.md]: In this tutorial we are going to show how to add UI5 Web Components to your application. UI5 Web Components can be added to both new Angular applications, as well as already existing ones.

File: docs/Vue-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Get started with UI5 Web Components & VueJS
+
+In this tutorial, we are going to show how to get add UI5 Web Components in your application. UI5 Web Components can be added to both new VueJS application as well as already existing one.
```
Comment (vladitasev) [inline docs/Vue-tutorial.md]: same as above

**Pull request merged**